docs/admin/pool_operations.md

```diff
@@ -290,6 +290,107 @@ The example below shows a rebuild in progress and NVMe space allocated.
+   Self-healing policy (self_heal)                                                  exclude
+   Rebuild space ratio (space_rb)                                                   0%
+   Pool service replica list (svc_list)                                             [0]
+   Pool service redundancy factor (svc_rf)                                          not set
+   Upgrade Status (upgrade_status)                                                  not started
+
+$ dmg pool upgrade pool1
+   Pool-upgrade command succeeded
+
+$ dmg pool get-prop pool1
+
+   Pool pool1 properties:
+   Name                                                                             Value
+   ----                                                                             -----
+   WAL Checkpointing behavior (checkpoint)                                          timed
+   WAL Checkpointing frequency, in seconds (checkpoint_freq)                        5
+   Usage of WAL before checkpoint is triggered, as a percentage (checkpoint_thresh) 50
+   EC cell size (ec_cell_sz)                                                        64 KiB
+   Performance domain affinity level of EC (ec_pda)                                 1
+   Global Version (global_version)                                                  1
+   Pool label (label)                                                               pool1
+   Pool performance domain (perf_domain)                                            root
+   Tier placement policy (policy)                                                   type=io_size
+   Pool redundancy factor (rd_fac)                                                  0
+   Reclaim strategy (reclaim)                                                       lazy
+   Performance domain affinity level of RP (rp_pda)                                 3
+   Checksum scrubbing mode (scrub)                                                  off
+   Checksum scrubbing frequency (scrub-freq)                                        604800
+   Checksum scrubbing threshold (scrub-thresh)                                      0
+   Self-healing policy (self_heal)                                                  exclude
+   Rebuild space ratio (space_rb)                                                   0%
+   Pool service replica list (svc_list)                                             [0]
+   Pool service redundancy factor (svc_rf)                                          2
+   Upgrade Status (upgrade_status)                                                  in progress
+```
+
+Duration of upgrade depends on possible format change across different DAOS releases.
+It might trigger rebuild to re-place objects data which might be time-consuming.
+So `dmg pool upgrade` will return within seconds, but it might take hours to see Pool Status
+become 'in progress' to 'completed'. pool will keep offline and deny any pool connection until
+pool upgrade status become 'completed'.
```

Comment:
```suggestion
Duration of upgrade depends on possible format change across different DAOS releases.
It might trigger rebuild to re-place objects data which might be time-consuming.
So `dmg pool upgrade` will return within seconds, but it might take hours to see Pool Status
change from 'in progress' to 'completed'. The pool will stay offline and deny any pool connection until
pool upgrade status become 'completed'.
```